xml/System/TimeOnly.xml

```diff
@@ -1556,11 +1623,13 @@
         <Parameter Name="result" Type="System.TimeOnly" RefType="out" />
       </Parameters>
       <Docs>
-        <param name="s">To be added.</param>
-        <param name="format">To be added.</param>
-        <param name="result">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="s">A string containing the characters representing a time to convert.</param>
+        <param name="format">The required format of s.</param>
+        <param name="result">When this method returns, contains the TimeOnly value equivalent to the time contained in s, if the conversion succeeded, or MinValue if the conversion failed. The conversion fails if the s is empty string, or does not contain a time that correspond to the pattern specified in format. This parameter is passed uninitialized.</param>
+        <summary>Converts the specified string representation of a time to its TimeOnly equivalent using the specified format and style.
+            The format of the string representation must match the specified format exactly. The method returns a value that indicates whether the conversion succeeded.</summary>
+        <returns>
+          <see langword="true" /> if s was converted successfully; otherwise, <see langword="false" />.</returns>
```

Comment:
```suggestion
        <param name="s">A string containing the time to convert.</param>
        <param name="format">The required format of <paramref name="s" />.</param>
        <param name="result">When this method returns, contains the <see cref="T:System.TimeOnly" /> value equivalent to the time contained in <paramref name="s" />, if the conversion succeeded, or <see cref="F:System.TimeOnly.MinValue" /> if the conversion failed. The conversion fails if <paramref name="s" /> is an empty string, or does not contain a time that correspond to the pattern specified in format. This parameter is passed uninitialized.</param>
        <summary>Converts the specified string representation of a time to its <see cref="T:System.TimeOnly" /> equivalent using the specified format and style.
            The format of the string representation must match the specified format exactly. The method returns a value that indicates whether the conversion succeeded.</summary>
        <returns>
          <see langword="true" /> if <paramref name="s" /> was converted successfully; otherwise, <see langword="false" />.</returns>
```